packages/api/app/testAuth.js

```diff
@@ -1,9 +1,18 @@
 'use strict';
 
-const { randomString } = require('@cumulus/common/test-utils');
+const { randomId } = require('@cumulus/common/test-utils');
 const get = require('lodash.get');
+const { createJwtToken } = require('../lib/token');
 
-let token = randomString();
+
+const newToken = () => {
+  let accessToken = randomId('oauthcode');
+  const username = 'testUser';
+  const expirationTime = new Date(Date.now() + 3600 * 24 * 1000);
```

Comment:
I'm bad at this. Is this a day from when it was created?

Reply:
yes.  60 sec/min * 60 min/hr * 24 hr/day * 1000 ms/sec.